src/domain/audio/InboundAudioStream.ts

```diff
@@ -140,7 +140,11 @@ class InboundAudioStream {
             console.debug("$$$$$$$ Ramp up volume of this message.");
 
             // TODO: Ramp up volume of this message - scale samples.
-
```

Comment:
Remove the "TODO" in the comment above, plus the console.debug() above it. Likewise, remove these in the ramping down code.